plugins/lighthouse/src/components/AuditList/AuditListForEntity.tsx

```diff
@@ -13,19 +13,65 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import React from 'react';
+import React, { ReactNode } from 'react';
 import { AuditListTable } from './AuditListTable';
 import { useWebsiteForEntity } from '../../hooks/useWebsiteForEntity';
-import { Progress } from '@backstage/core-components';
+import { LIGHTHOUSE_WEBSITE_URL_ANNOTATION } from '../../../constants';
+import {
+  Content,
+  ContentHeader,
+  InfoCard,
+  Progress,
+  WarningPanel,
+} from '@backstage/core-components';
+import { Button } from '@material-ui/core';
+import { resolvePath, useNavigate } from 'react-router-dom';
+import { useEntity } from '@backstage/plugin-catalog-react';
+import { useRouteRef } from '@backstage/core-plugin-api';
+import { rootRouteRef } from '../../plugin';
+import LighthouseSupportButton from '../SupportButton';
 
 export const AuditListForEntity = () => {
   const { value, loading, error } = useWebsiteForEntity();
+  const { entity } = useEntity();
+  const navigate = useNavigate();
+  const fromPath = useRouteRef(rootRouteRef)?.() ?? '../';
```

Comment:
Hmm we should really update this plugin to use `subRouteRefs` for these paths instead of this, would make all this code a little cleaner. But maybe don't have to do that in this PR, would be good as a follow up.

Reply:
@benjdlambert can you clarify which paths should be changed to `subRouteRefs`? In this component, the resolved path for the `create-audit` button will change depending on the websiteUrl of the entity. Correct me if I'm wrong, but I don't think we can statically create a `subRouteRef` for these audit links in `Router.tsx` due to the dynamic nature of this link.

Reply:
We could have parameters based on the `subRouteRef` though. Which we can use to do the routing to the `create-audit` places and other routes that the  plugin uses. It would just clean up some of the routing logic here I think.